suite/tests/api/opnd-a64.c

```diff
@@ -81,17 +83,19 @@ test_get_size()
             :
             : "x0");
         opsz_vl = opnd_size_from_bytes(vl);
+        opsz_pl = opnd_size_from_bytes(vl / 8);
```

Comment:
Please use a more descriptive name: I don't know what "pl" means.  (I would also prefer "veclen" to "vl": I think more users will understand that.)

Reply:
Done.